command/sync.go

```diff
@@ -460,7 +459,7 @@ func (s Sync) planRun(
 	}()
 
 	// only in destination
-	if s.delete {
+	if s.delete && len(onlyDest) > 0 {
```

Comment:
Thanks for fixing the problem.

onlyDest is a chan object here (it was a slice in the previous version). So I'm not sure if this check actually does what is desired. Shouldn't we check the length of the dstURLs inside the if block?